cli/commands/run/flags.go

```diff
@@ -29,6 +29,7 @@ const (
 	UnitsThatIncludeFlagName               = "units-that-include"
 	DependencyFetchOutputFromStateFlagName = "dependency-fetch-output-from-state"
 	UsePartialParseConfigCacheFlagName     = "use-partial-parse-config-cache"
+	VersionCacheFileFlagName               = "version-cache-file"
```

Comment:
Could we change this flag name to something like `version-manager-file-name`?

I think, generally, when we use the `-file` prefix, we're specifying the path to a file on disk. This is specifying the base name for version manager files, which could be spread out throughout a project. It's also not controlling the name of a file that would cache versions, which is what I interpret the flag to mean reading it naively.